### DISCUSSION TITLE
unable to get the velero metrics in prometheus

### DISCUSSION THREAD

**Discussion by sheril5** [vmware-tanzu/velero#6358] [Community support Q&A]
i need to get the metrics of velero in prometheus but i was unable to get it.i have also enabled the servicemonitor but still i am facing this issue

**Comment by WRKT** [comment]
Facing the same issue right now, enable serviceMonitor, but when I check my prometheus server, there is no velero metrics.
But if I do a `curl` in a pod on the pod service /metrics, i have the list of all metrics...

**Reply by sotolko** [reply]
@WRKT facing the same issue, i can see metrics with curl in a pod service /metrics, but cant on prometheus, did you find any solution on this?
